apps/web/package.json

```diff
@@ -10,15 +10,18 @@
     "test": "npx cypress verify & cypress run --component"
   },
   "dependencies": {
+    "@heroicons/react": "^2.1.1",
+    "@tailwindcss/forms": "^0.5.7",
+    "@types/lodash": "^4.14.202",
     "axios": "^1.6.7",
+    "clsx": "^2.1.0",
+    "formik": "^2.4.5",
+    "lodash": "^4.17.21",
     "next": "14.0.4",
     "react": "^18",
     "react-dom": "^18",
+    "react-toastify": "4.1",
     "use-debounce": "^10.0.0",
-    "@heroicons/react": "^2.1.1",
```

Comment:
tailwindcss/forms, clsx, and formik are needed. Why removed?

Reply:
- means removed, + means added. if there's a + and a - in the same files with the same value it means only the order of the line is changed, and it's not removed. if you carefully read there's: 
`+"@tailwindcss/forms": "^0.5.7"` at line 14
`+"clsx":"^2.1.0"`, at line 17
`+"formik":"^2.4.5"` at line 18.